### ISSUE TITLE
OER Assignment Instructions - Help Please? 

### ISSUE THREAD

**Issue by NurseKillam** [sparcopen/doathon#47]
[//]: # "=======================
Even if you know Github well, we suggest you read this. Anything between these lines you can leave or delete, as they won't display anyway when you post (you can check this via Preview changes). They're here to help you complete issues quickly and in a way that will help other participants. If you're posting a new project, or challenge. We suggest you fill out the Google Forms first.
============================"

I am working to create more OER in nursing with students - making it one assignment option. To make their experience better I am looking for feedback. 

[//]: # "=======================
Please paste the metadata you received after submitting your project or challenge in your Google Form exactly as we sent it to you. You can delete what's there now, it's just there
============================"

* **Submission Name:** OER Assignment Instructions
* **Contact Lead:** @NurseKillam on Twitter
* **External Site Link (optional - primarily for projects):** http://open.nursekillam.com/
* **Region**: #NorthernAmerica Ontario, Canada 
* **Issue Area:** #OpenEducation #OpenResearch
* **Issue Type:** Project 
* **Types of Support Needed:** I thought I could get some feedback on assignment instructions and rubrics that I am currently developing. It is part of a related self-study. 

## Description
I am a teacher who has embraced the principles of OER long before hearing about it in August. As I engage in my first true OER assignment for students I wanted to seek feedback from others on my assignment instructions. Students have the option of working on OER or completing a different assignment inside their learning management system. During the process I will be doing a self-study of my experience as described under the "research" tab. 

[//]: # "=======================
Insert a paragraph providing more context for your project or challenge focuses on. For project leads, this is a good place to give some broader context about your project—beyond the scope of the do-a-thon. If you're posting a challenge, this is a good chance to say how the problem arise or why it feels relevant to you.
============================"

#Feedback #Research 

## How can others contribute? 
I am new to GitHub, but we can discuss here. Twitter also works. I am in the process of seeing if I can turn comments on the website on for each page. 
I only speak English fluently. The website will be used in January. 

[//]: # "======================= 
Please say what the best way to contribute to the project or challenge is, sometimes that will just be 'lets discuss here' or 'Ive started a Google doc'. 
If you are a challenge lead, give some context on what design thinking tools you will be using, and how other folks can update their ideas onto the thread. 
If you are a project lead, and you already have clear ways people can contribute it might be worth linking to them here.
Language: If your project is regionally based in a non-English-speaking region, clarify here what language you and contributors will primarily be communicating in. 
If you're leading a project or challenge participating remotely: Use this space to let people know that this is a remote project and that you are not 'in the room' in Berlin. Let other participants know what the best way to get in touch with you, where the work will happen, and where any updates or outputs will go. 
If you are at the in-person meeting in Berlin: Be as inclusive as possible to those outside the room.  Use this space to give clear instructions to those participating in the do-a-thon remotely on how they can keep up to date and contribute. ============================"

[//]: # "=======================
You're ready to post!!! After posting your issue, the real work begins. Next, you might want to:
   Tweet a link to this issue with #opencon so others can join in
   Make another issue to involve people in your work - remember to use your metadata
   Come back from time to time and update the community on your project.
You'll get an email update whenever someone interacts with your issue.
============================"

_This post is part of the OpenCon 2017 Do-A-Thon. Not sure what's going on? [Head here](http://doathon.opencon2017.org/)._

**Comment by Daniel-Mietchen** [comment]
I had a quick look and am wondering whether the images on the site are all yours - if not, their source and license should be indicated.

**Comment by NurseKillam** [comment]
Good point. They are all from Pixabay. I used the plugin so they should be cited when you go to the page (just under the image).

**Comment by NurseKillam** [comment]
I am not sure why it isn't showing up.
